extensions/ArchLinux/modules/skins/timeless.less

```
@@ -0,0 +1,96 @@
+@import '../arch_definitions';
+
+/* disable default mediawiki logo  */
+#p-logo {
+    display: none !important;
+}
+
+/* make room for archnavbar and disable fixed searchbar */
+body, #mw-content-container {
+    margin-top: 0;
+    background-color: @body-background-color;
+}
+#mw-content-container {
+    border-bottom: none;
+    padding-top: .5em;
+}
+#mw-header-hack, #mw-header-nav-hack {
+    display: none !important;
+}
+#mw-header-container {
+    position: initial;
+}
```

Let's make the header blue:

```suggestion
#mw-header-container {
    position: initial;
    background-color: @body-background-color;
}
```

---

All right.